lib/bolt/error.rb

```diff
@@ -50,19 +50,32 @@ def initialize(msg)
   class RunFailure < Error
     attr_reader :result_set
 
-    def initialize(result_set, action, object)
+    def generate_msg(result_set, action, object)
```

Comment:
I think we can just override `msg` and store the and grab the action/object/result_set from details

Reply:
Overrode `to_s`, as that seems to be what everything else - inspect, message, msg - uses.